### DISCUSSION TITLE
Welcome to SympyLab Discussions!

### DISCUSSION THREAD

**Discussion by zwang20** [zwang20/SympyLab#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by edward-ji** [comment]
Hello, I am Edward Ji. I am very interested in this project because I am doing a lot of maths recently. I need a tool to do some tedious and repetitive calculations for me. 😄 Michael and I are previously developing [YEquals](https://github.com/Quark-Games/YEquals) - an unsafe and poorly implemented grapher. We are hoping to improve on that.
